webrtc.html

```diff
@@ -7283,25 +7283,24 @@ <h2>Methods</h2>
       <p>The <dfn>RTCRtpContributingSource</dfn> and
       <dfn>RTCRtpSynchronizationSource</dfn> dictionaries contain information
       about a given contributing source (CSRC) or synchronization source (SSRC)
-      respectively, including the most recent time a
-      packet that the source contributed to was played out. The browser MUST
-      keep information from RTP packets received in the previous 10 seconds.
-      When the first audio or video frame contained in an RTP packet is
-      delivered to the <code><a>RTCRtpReceiver</a></code>'s
-      <code><a>MediaStreamTrack</a></code> for playout, the user agent MUST
-      queue a task to update the relevant information for the
-      <code><a>RTCRtpContributingSource</a></code> and
-      <code><a>RTCRtpSynchronizationSource</a></code> dictionaries based on the
-      contents of the packet. The information relevant to the
-      <code><a>RTCRtpSynchronizationSource</a></code> dictionary corresponding
-      to the SSRC identifier, is updated each time, and if the RTP packet
-      contains CSRC identifiers, then the information relevant to the
-      <code><a>RTCRtpContributingSource</a></code> dictionaries corresponding to
-      those CSRC identifiers is also updated.</p>
+      respectively. When an audio or video frame from an RTP packet is delivered
+      to the <code><a>RTCRtpReceiver</a></code>'s
+      <code><a>MediaStreamTrack</a></code>, the user agent SHOULD queue a task
```

Comment:
Don't ever use SHOULD if you can avoid it. It's impossible to write a good test for a SHOULD, because the conditions under which the UA can skip it are not precisely defined. (If they had been, it would have been a "MUST if....")

Reply:
Ack on that. I'll change the SHOULD here to MUST. It just felt strange to have a MUST followed by a note that immediately contradicts the usual RFC definition of MUST.